presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++                                          +                                          +                                    +``resource_overcommit``. The session may also want to  +
++                                          +                                          +                                    +decrease system pressure, so it's possible to decrease +
++                                          +                                          +                                    +query memory limit for session by setting              +
++                                          +                                          +                                    +``query_max_memory`` to smaller value. Setting         +
++                                          +                                          +                                    +``query_max_memory`` to higher value then              +
++                                          +                                          +                                    +``query.max-memory`` will not have any effect. This    +
++                                          +                                          +                                    +property may be used to ensure that single query       +
++                                          +                                          +                                    +cannot use all resources in cluster. The value should  +
++                                          +                                          +                                    +be set to be a little bit over what typical expected   +
++                                          +                                          +                                    +query in system will need - that way system will be    +
++                                          +                                          +                                    +resistant to SQL bugs that would cause large unwanted  +
++                                          +                                          +                                    +large computation. Also if rare usecases will require  +
++                                          +                                          +                                    +more memory, then the ``resource_overcommit`` session  +
++                                          +                                          +                                    +property may be used to break the limit.               +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++      ``query.max-memory-per-node``       +          ``String`` (data size)          +              ``1 GB``              +The purpose of that is same as of ``query.max-memory`` +
```

Comment:
As this have almost same consequences as ``query.max-memory`` I would just leave it as reference and extend ``query.max-memory`` to cover that.